src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java

```diff
@@ -94,6 +94,12 @@ public void testForUniqueIDsInGeneratedRestDSLOutput() throws Exception {
 		assertEquals(0, placeholders);
 	}
 
+	@Test
+	public void testForRestDSLOutputFromSimpleOpenAPIFile() throws Exception {
+		String camelContextXML = getRestDSLXMLFromSwagger("src/test/resources/simple-swagger.json");
+		assertTrue(camelContextXML != null && camelContextXML.trim().length() > 0);
```

Comment:
having a single assertion will make it harder to investigate instead of having 2 assertions. having 2 assertions will point directly if the value is null or if it is an empty one.